plugins/techdocs-backend/src/service/router.ts

```diff
@@ -287,11 +322,17 @@ export async function createRouter(
   // Route middleware which serves files from the storage set in the publisher.
   router.use('/static/docs', publisher.docsRouter());
 
-  return router;
-}
+  // Endpoint that sets the cookie for the user
+  router.get('/cookie', async (req, res) => {
+    const credentials = await httpAuth.credentials(req, {
+      allow: ['user'],
+      allowLimitedAccess: true,
```

Comment:
Just to confirm, are these options correct?

Reply:
With the current `issueUserCookie` implementation we should be able to skip this step and call `issueUserCookie` without a credentials option. It'll pick up the credentials from the request if available, but also translate to a noop for the old system.

Reply:
That works, thank you 🙏🏻

Reply:
Oh I completely forgot that that's what it does!

Reply:
Can we really skip adding a router policy for `/cookie` letting through `'unauthenticated'` though?